LICENSE.rst

```diff
@@ -0,0 +1,30 @@
+BSD 3-Clause License
+
+Copyright (c) 2009-2024, The Trustees of Columbia University
```

Comment:
2009-2024 added

Reply:
I think this work does not go back to 2009